components/payments/payment-modal.js

```diff
@@ -5,6 +5,7 @@ import Modal from "../modal"
 export default function AddPaymentModal({ showModal, setShowModal, addNewPayment }) {
   const merchantNameInput = useRef()
   const acctNumInput = useRef()
+  const expireDateInput = useRef()
```

Comment:
I'd suggest using state to manage these instead of ref as per steves reccomendations

Reply:
i agree with this, however i find that if i were to use state here that would require rewriting the rest of these to keep consistency, but if that's the standard the team wants to keep ill rewrite these too 👍